src/pages/home/report/ReportActionCompose/ComposerWithSuggestions.js

```diff
@@ -446,7 +486,12 @@ function ComposerWithSuggestions({
             }
 
             focus();
-            replaceSelectionWithText(e.key, false);
+            // Reset cursor to last known location
+            setSelection((prevSelection) => ({
+                start: prevSelection.start + 1,
+                end: prevSelection.end + 1,
```

Comment:
@mollfpr  Earlier it had to handle both Emoji picker input and Keyboard onPress. When emoji was inserted via picker selection, it had to adjust for emoji with white space. Now since the logic is moved into key press event, the selection is always going to be incremented by one, since it gets triggered only once for the first character being inserted.